docs/design/node-selector.md

```diff
@@ -0,0 +1,37 @@
+## Introduction
+
+for example, my k8s cluster has 10 nodes (10 gpu per node), 5 for training, 5 for serving, I want to use volcano schedule tfjob/pytorchjob/vcjob on training nodes. use default-scheduler schedule online pod on serving nodes. it is work by setting nodeSelector or nodeAffinity to pod, but it is not properly when considering volcano queue mechanism.
+
+if there are two queue: queue1(weight=1) and queue2 (weight=1), it is overused if queue1 used 40 gpu, but it's weight resource is 50 (100 * 1/2), so if queue2 want use 20 gpu, reclaimable mechanism will not work.
```

Comment:
If there are totally 100 GPUs in the cluster, the deserved resource for queue1 and queue2 are both 50 GPUS. Why is it overused if queue1 used 40 GPUs? BTW, please make the format and grammar of the doc more canonical.